.github/workflows/ci.yml

```diff
@@ -32,6 +32,9 @@ jobs:
           - caliban.nixos.org
           - elated-minsky
           - sleepy-brown
+          - haumea
```

Comment:
Is the comment above (line 26) now vestigial?

> all builds combined consume too much disk space... we should soon switch to something else

(not a blocker) Could/should we be doing something else to generate this list of machines so we don't have to remember to keep it up to date as we add/remove/rename machines? Is that what https://github.com/nix-community/nix-github-actions is for?

Reply:
nix-github-actions would also work. But the current solution works actually good enough and works for github forks. nix-fast-build will only rebuild machines if there are actually changes. So we might just keep it this way for simplicity.

Reply:
Sounds good. Feel free to mark this resolved once the comment on line 26 is updated (if it needs to be updated).